CHANGELOG.md

```diff
@@ -344,6 +367,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [0.3.1]: https://github.com/square/Blueprint/compare/0.3.0...0.3.1
 [0.3.0]: https://github.com/square/Blueprint/compare/0.2.2...0.3.0
 [0.2.2]: https://github.com/square/Blueprint/releases/tag/0.2.2
+[#135]: https://github.com/square/Blueprint/pull/135
```

Comment:
What is this list for? Is every merged PR supposed to be here?

Reply:
Markdown supports [two styles of links][markdown links]: inline and reference. These are the references for reference links. (The links that use these are [shortcut reference links], which is a Github addition that makes `[foo]` shorthand for `[foo][foo]`, which is why the release headers like `[0.13.0]` work as links.)

I prefer this style for PR links because it makes the text more readable, but it's not a requirement. They _are_ required for the version links to work, though.

[markdown links]: https://daringfireball.net/projects/markdown/syntax#link
[shortcut reference links]: https://github.github.com/gfm/#shortcut-reference-link

Reply:
Ahh, I thought it was automatically detecting those because ✨github✨

Reply:
I wish!